libs/community/langchain_community/document_loaders/directory.py

```diff
@@ -69,6 +71,7 @@ def __init__(
             loader_kwargs = {}
         self.path = path
         self.glob = glob
+        self.exclude = exclude
```

Comment:
I'd suggest doing a run time check here to verify that exclude is actually a list. The variable name sounds like a string, so likely API usage will be a mixture of sequence[str] as well as str from users. With a run time check, it will be easier to surface an error to users that's informative

Reply:
:+1: same question as above: should I add it only here or in the other loader as well?

https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/document_loaders/blob_loaders/file_system.py#L111

Reply:
I went ahead and just added this here. It was easy to just handle both, so I made strings an option as well and added a test.

Reply:
Thank you!